ui/app/components/clients/history.js

```diff
@@ -89,10 +90,81 @@ export default class History extends Component {
     );
   }
 
-  get latestUpgradeData() {
-    // {id: '1.9.0', previousVersion: null, timestampInstalled: '2021-11-03T10:23:16Z'}
-    // version id is 1.9.0 or earliest upgrade post 1.9.0, timestamp is RFC3339
-    return this.args.model.versionHistory[0] || null;
+  get upgradeVersionHistory() {
+    const versionHistory = this.args.model.versionHistory;
+    if (!versionHistory || versionHistory.length === 0) {
+      return null;
+    }
+
+    // get upgrade data for initial upgrade to 1.9 and/or 1.10
+    let relevantUpgrades = [];
+    const importantUpgrades = ['1.9', '1.10'];
+    importantUpgrades.forEach((version) => {
+      let findUpgrade = versionHistory.find((versionData) => versionData.id.match(version));
+      if (findUpgrade) relevantUpgrades.push(findUpgrade);
+    });
+
+    // if no history for 1.9 or 1.10, get first upgrade available
+    if (Object.entries(relevantUpgrades).length === 0) {
```

Comment:
same comments. I wonder if this logic is useful as a helper/util fn... 🤔

Reply:
That's a good suggestion, for the whole getter?  Or just line 108 - I'll make a note and pulling it out may be a good call come test-writing time.